libs/cardano-ledger-api/test/Tests.hs

```diff
@@ -25,9 +25,10 @@ apiSpec =
     describe "State" $ do
       StateQuery.spec
     describe "Imp" $
-      forM_ [natVersion @9, natVersion @10] $ \v ->
-        withImpStateWithProtVer @Conway v $ do
-          ImpQuery.spec @Conway
+      withImpInit @(LedgerSpec Conway) $
+        forM_ [eraProtVerLow @Conway .. eraProtVerHigh @Conway] $ \v ->
```

Comment:
I've seen this pattern in a couple of places, maybe this should be a function

Reply:
Added a reusable `eraProtVersions` function.